python-sdk/tutorials/automl-with-azureml/forecasting-pipelines/auto-ml-forecasting-pipelines.ipynb

```diff
@@ -0,0 +1,783 @@
+    "metrics_output_name = \"metrics_output\"\n",
+    "best_model_output_name = \"best_model_output\"\n",
+    "model_file_name = \"model_file\"\n",
+    "\n",
+    "metrics_data = PipelineData(\n",
+    "    name=\"metrics_data\",\n",
+    "    datastore=datastore,\n",
+    "    pipeline_output_name=metrics_output_name,\n",
+    "    training_output=TrainingOutput(type=\"Metrics\"),\n",
+    ")\n",
+    "model_data = PipelineData(\n",
+    "    name=model_file_name,\n",
+    "    datastore=datastore,\n",
+    "    pipeline_output_name=best_model_output_name,\n",
+    "    training_output=TrainingOutput(type=\"Model\"),\n",
+    ")\n",
+    "\n",
+    "automl_step = AutoMLStep(\n",
+    "    name=\"automl_module\",\n",
+    "    automl_config=automl_config,\n",
+    "    outputs=[metrics_data, model_data],\n",
+    "    allow_reuse=False,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Register Model Step\n",
+    "Script to register the model to the workspace. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.pipeline.core import PipelineData\n",
+    "\n",
+    "# The model name with which to register the trained model in the workspace.\n",
+    "model_name_str = \"ojmodel\"\n",
+    "model_name = PipelineParameter(\"model_name\", default_value=model_name_str)\n",
+    "\n",
+    "\n",
+    "register_model_step = PythonScriptStep(\n",
+    "    script_name=\"register_model.py\",\n",
+    "    name=\"register_model\",\n",
+    "    source_directory=\"scripts\",\n",
+    "    allow_reuse=False,\n",
+    "    arguments=[\n",
+    "        \"--model_name\",\n",
+    "        model_name,\n",
+    "        \"--model_path\",\n",
+    "        model_data,\n",
+    "        \"--ds_name\",\n",
+    "        \"dominicks_OJ_train\",\n",
+    "    ],\n",
+    "    inputs=[model_data],\n",
+    "    compute_target=compute_target,\n",
+    "    runconfig=conda_run_config,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Build the Pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline = Pipeline(\n",
+    "    description=\"training_pipeline\",\n",
+    "    workspace=ws,\n",
+    "    steps=[automl_step, register_model_step],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Submit Pipeline Run"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline_run = experiment.submit(training_pipeline)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "training_pipeline_run.wait_for_completion(show_output=False)"
+   ]
+  },
```

Comment:
Could we download the metrics from pipeline output and demonstrate it to user?

Reply:
added